pkgs/development/tools/vend/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "vend";
+  version = "unstable-2020-06-04";
+
+  # A permanent fork from master is maintained to avoid non deterministic go tidy
+  src = fetchFromGitHub {
+    owner = "c00w";
+    repo = "vend";
+    rev = "49899829b1e4bcd22ee3ba2d12bc76863196cb03";
+    sha256 = "1c597wcc6avzpavad3m2dj5vzvqbwrrpb6z7wgff8dc341xr62cj";
+  };
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/c00w/vend";
+    description = "A utility which vendors go code including c dependencies.";
```

nitpick, `description` shouldn't contain periods
```suggestion
    description = "A utility which vendors go code including c dependencies";
```

---

Done